arch/lkl/kernel/time.c

```diff
@@ -29,6 +29,11 @@ void calibrate_delay(void)
 {
 }
 
+void read_persistent_clock(struct timespec *ts)
```

Comment:
thanks for the comment @Rondom.

> It might make sense to make this public i.e. in addition add some helper function in user space using settimeofday/adjtimex that syncs the host time with the lkl time.

I don't understand this comment; what do you mean by making a function public ?  btw, 66513c97061bb9c2b2a280b76b1dbdbf4b145ddf exposes all kernel functions by putting a prefix to avoid symbol collision.
(see https://github.com/lkl/linux/issues/17#issuecomment-158690204 more detail)


though you might be aware of this, this additional function overrides a function of kernel (below), which each architecture *can* define its own way to access to a clock.

`void __weak read_persistent_clock(struct timespec *ts)` in kernel/time/timekeeping.c.